website/assets/js/pages/entrance/signup.page.js

```diff
@@ -12,21 +12,18 @@ parasails.registerPage('signup', {
 
     // Form rules
     formRules: {
-      fullName: {required: true},
+      firstName: {required: true},
+      lastName: {required: true},
+      organization: {required: true},
       emailAddress: {required: true, isEmail: true},
-      password: {required: true},
-      confirmPassword: {required: true, sameAs: 'password'},
-      agreed: {required: true},
+      password: {required: true, minLength: 8},
     },
-
     // Syncing / loading state
     syncing: false,
-
     // Server error state
     cloudError: '',
-
+    showFullForm: false,
```

Comment:
Since `showFullForm` is probably not part of the server error state, I think it should be separated by newlines for clarity